### PR TITLE
Feature/josecaste new article

### PR DESCRIPTION
**Pull request by JoseCaste**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -64,18 +70,53 @@
...
+			<artifactId>firebase-admin</artifactId>
+			<version>${firebase.version}</version>
 		</dependency>
 		<dependency>
-				<groupId>org.springframework.boot</groupId>
-				<artifactId>spring-boot-starter-tomcat</artifactId>
-				<scope>provided</scope>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-tomcat</artifactId>
+			<!--<scope>provided</scope>-->
```
Comment (JoseCaste) [inline pom.xml:82]: I left this commented code because I had application deploying issues. If a deleted this one, the run will fail

Comment (cesar-martinezg) [inline pom.xml:82]: remove unused code

File: pom.xml [hunk 2]
```diff
@@ -64,18 +70,53 @@
...
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-api</artifactId>
+			<scope>test</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-all</artifactId>
+			<version>1.10.19</version>
```
Comment (cesar-martinezg) [inline pom.xml:104]: set dependencies version on properties at the beginning of the pom file

File: src/main/java/com/softserve/exceptions/ForgotPasswordProcess.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package com.softserve.exceptions;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode(callSuper = false)
+public class ForgotPasswordProcess extends Exception{
```
Comment (cesar-martinezg) [inline src/main/java/com/softserve/exceptions/ForgotPasswordProcess.java:8]: Rename to "ForgotPasswordProcess**Exception**"

File: src/main/java/com/softserve/util/Email.java [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+import org.springframework.ui.freemarker.FreeMarkerTemplateUtils;
+
+import com.softserve.dto.EmailContent;
+
+import freemarker.template.Configuration;
+import freemarker.template.Template;
+import freemarker.template.TemplateException;
+
+@Service
+public class Email {
```
Comment (cesar-martinezg) [inline src/main/java/com/softserve/util/Email.java:26]: Rename to EmailService

**Review by cesar-martinezg — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (cesar-martinezg) [reply pom.xml:82]: so, when committing, revert the change, then you can apply again in your local

**Review by cesar-martinezg — APPROVED** [review]

**Pull request merged**